src/api.ts

```diff
@@ -0,0 +1,114 @@
+interface Sample {
+  folder: {
+    id: number;
+    name: string;
+    owner: {
+      id: number;
+      name: string;
+      profileImageSource: string;
+    };
+    links: Link1[];
+  };
+}
+
+interface User {
+  id: number;
+  name: string;
+  email: string;
+  profileImageSource: string;
+}
+
+interface UserInfo {
+  data: [
+    {
+      id: number;
+      created_at: string;
+      name: string;
+      image_source: string;
+      email: string;
+      auth_id: string;
+    }
+  ];
+}
```

Comment:
```suggestion
  data: {
    id: number;
    created_at: string;
    name: string;
    image_source: string;
    email: string;
    auth_id: string;
  }[];
```

`Array<T>` 혹은 `T[]` 와 `[T]`는 다르니 주의해주세요~!